Makefile

```diff
@@ -66,8 +66,9 @@ pdf: latex
 	mv $(BUILDDIR)/latex/$(LANG)/PyQGISDeveloperCookbook.pdf $(BUILDDIR)/pdf/$(LANG)/;
 	mv $(BUILDDIR)/latex/$(LANG)/QGISTrainingManual.pdf $(BUILDDIR)/pdf/$(LANG)/;
 
-site: html
+site: html pdf
 	rsync -az $(BUILDDIR)/html/$(LANG) $(SITEDIR)/
+	rsync -az $(BUILDDIR)/pdf $(SITEDIR)/
```

Comment:
@rduivenvoorde, can you confirm this is right to pull docs on the server? Thanks

Reply:
just try...

Reply:
Do you mean in https://github.com/qgis/QGIS-Documentation/pull/5729#issuecomment-647112293 that we should not do this?